src/prefect/tasks/jupyter/jupyter.py

```diff
@@ -19,7 +19,10 @@ class ExecuteNotebook(Task):
         - parameters (dict, optional): dictionary of parameters to use for the notebook
             Can also be provided at runtime
         - output_format (str, optional): Notebook output format.
+            This argument is gnored if `exporter` is provided.
```

Comment:
Typo (if we keep the interface as implemented)

```suggestion
            This argument is ignored if `exporter` is provided.
```